dev.sh is broken

Hi all,

I found `dev.sh` today and it seems very useful to ease contributions, however when executing the script I got this error:

```
Cloning into '../strider-ecosystem-client'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
./dev.sh: line 25: pushd: ../strider-ecosystem-client: No such file or directory
./dev.sh: line 27: popd: directory stack empty
npm ERR! Darwin 15.5.0
npm ERR! argv "node" "/Users/.../.nvm/v0.10.32/bin/npm" "link" "../strider-ecosystem-client"
npm ERR! node v0.10.32
npm ERR! npm  v3.8.1
npm ERR! code EINVALIDTYPE

npm ERR! typeerror Error: Argument #2: Expected array but got string
npm ERR! typeerror     at EventEmitter.install (/Users/alfredolopezmolto/.nvm/v0.10.32/lib/node_modules/npm/lib/install.js:175:3)
npm ERR! typeerror     at Object.commandCache.(anonymous function) (/Users/alfredolopezmolto/.nvm/v0.10.32/lib/node_modules/npm/lib/npm.js:216:13)
npm ERR! typeerror     at /Users/alfredolopezmolto/.nvm/v0.10.32/lib/node_modules/npm/lib/link.js:89:24
npm ERR! typeerror     at Object.oncomplete (fs.js:107:15)
npm ERR! typeerror This is an error with npm itself. Please report this error at:
npm ERR! typeerror     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
```

If I print `DEPS` then got:
`strider-bitbucket strider-build-badge strider-cli strider-custom strider-ecosystem-client strider-email-notifier strider-env strider-extension-loader strider-git strider-github strider-gitlab strider-heroku strider-mailer strider-metadata strider-node strider-python strider-ruby strider-simple-runner strider-slack strider-ssh-deploy`

The problem is the repo is not called *strider-ecosystem-client*, its name is *ecosystem-client* (https://github.com/Strider-CD/ecosystem-client).

Is it possible to rename the project to make this script work or is it better to make a specific patch in the script?

Thanks you in advance,

Alfredo

--- a/dev.sh
+++ b/dev.sh
@@ -16,11 +16,14 @@ BASE="git@github.com:Strider-CD"
 
 mkdir -p node_modules
 
-
 for module in $DEPS
 do
+  if [ $module == "strider-ecosystem-client" ]; then
+    module="ecosystem-client"
+  fi
+  repo="$module.git"
   if [ ! -d ../$module ]; then
-    git clone $BASE/$module ../$module
+    git clone $BASE/$repo ../$module
   fi
   pushd ../$module
   npm i

